decidim-proposals/app/commands/decidim/proposals/endorse_proposal.rb

```diff
@@ -35,6 +40,19 @@ def build_proposal_endorsement
         end
         endorsement
       end
+
+      def notify_endorser_followers
+        recipient_ids = @proposal.author.followers.pluck(:id)
+        Decidim::EventsManager.publish(
+          event: "decidim.events.proposals.proposal_endorsed",
+          event_class: Decidim::Proposals::ProposalEndorsedEvent,
+          resource: @proposal,
+          recipient_ids: recipient_ids.uniq,
+#            extra: {
```

Comment:
Remove this?

Reply:
why? it implements this: https://github.com/decidim/decidim/issues/2287#issuecomment-356944870

Reply:
Ah ok, I think you meant the commented code. Yes, it is done.